### PR TITLE
Add SqlClient Tests to Github

### PR DESCRIPTION
**Pull request by corivera**
The SqlClient test suite in ManualTests is run by hand using msbuild, and is thus marked as unsupported on all platforms to prevent auto runs. Connection strings also need to be added in DataCommon\ConnectionString.xml for the targeted test servers. They use the Microsoft sample databases "Northwind" or "Pubs", and each connection string references which version of SQL server it's used to test against.

### CODE REVIEW

**Comment by corivera** [comment]
CC @saurabh500 @kkurni @nadiatk

File: src/System.Data.SqlClient/tests/ManualTests/SQL/SqlRandom/SqlRandomTable.cs [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    /// <summary>
+    /// represents table with random column types and values in it
+    /// </summary>
+    public class SqlRandomTable
```
Comment (saurabh500) [inline src/System.Data.SqlClient/tests/ManualTests/SQL/SqlRandom/SqlRandomTable.cs]: Is this class used anywhere?  I don't find references of the Constructor outside the Create method of this class. 
And I don't see the reference to Create anywhere either.

**Comment by saurabh500** [comment]
This is a port with many files. I couldn't go through all the files.
I see some classes which I didn't find getting used in the tests directly or indirectly. However they look like they can be used in future, or were part of other tests which may not have been brought in.
Even If they are not used, don't remove those from the PR.

Why did you choose to make a single projects instead of multiple projects for this port?

Overall LGTM.

**Comment by corivera** [comment]
Each test is run from an assembly, so if all of these were separate projects you would need a different assembly for each. I found it much easier to organize and use the tests if they were just in one "ManualTests" assembly. Plus it reduces the number of files you need to maintain, since there aren't ~20 csproj and project.json files.

**Comment by saurabh500** [comment]
Sounds good. Thanks

**Comment by kkurni** [comment]
LGTM

**Pull request merged**